### PR TITLE
Added some actions about speaking ...

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

File: openslides_backend/action/speaker/create_update_delete.py [hunk 1]
```diff
@@ -17,3 +24,8 @@ class SpeakerActionSet(ActionSet):
     )
     update_schema = DefaultSchema(Speaker()).get_update_schema(["marked"])
     delete_schema = DefaultSchema(Speaker()).get_delete_schema()
+    routes = {
```
Comment (jsangmeister) [inline openslides_backend/action/speaker/create_update_delete.py:27]: The syntax here has changed, see https://github.com/OpenSlides/openslides-backend/pull/205

File: openslides_backend/action/speaker/create_update_delete.py [hunk 2]
```diff
@@ -1,9 +1,16 @@
 from ...models.models import Speaker
 from ..action_set import ActionSet
 from ..default_schema import DefaultSchema
+from ..generics import CreateAction, DeleteAction, UpdateAction
 from ..register import register_action_set
 
 
+class SpeakerDeleteAction(DeleteAction):
```
Comment (jsangmeister) [inline openslides_backend/action/speaker/create_update_delete.py:8]: This class seems unnecessary?

File: openslides_backend/action/speaker/speak.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+from ...shared.exceptions import ActionException
+from ...shared.interfaces import Event
+from ...shared.patterns import Collection, FullQualifiedId
+from ..base import Action, ActionPayload, DataSet, WriteRequestElement
+from ..default_schema import DefaultSchema
+from ..register import register_action
+
+
+@register_action("speaker.speak")
+class SpeakerSort(Action):
```
Comment (jsangmeister) [inline openslides_backend/action/speaker/speak.py:15]: Please rename.

File: openslides_backend/action/speaker/speak.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    """
+
+    model = Speaker()
+    schema = DefaultSchema(Speaker()).get_default_schema(
+        required_properties=["id"],
+        title="Speak schema",
+        description="Schema to let a speaker's speach begin.",
+    )
+
+    def prepare_dataset(self, payload: ActionPayload) -> DataSet:
```
Comment (jsangmeister) [inline openslides_backend/action/speaker/speak.py:27]: I suppose if a LOS is closed one cannot speak on it? If yes, this should be checked and tested here.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/speaker/create_update_delete.py (hunk 1 above)
Comment (normanjaeckel) [reply openslides_backend/action/speaker/create_update_delete.py:27]: Done.

File: openslides_backend/action/speaker/create_update_delete.py (hunk 2 above)
Comment (normanjaeckel) [reply openslides_backend/action/speaker/create_update_delete.py:8]: At the moment we do not have internal actions so I need this class if I do not want to register it.

File: openslides_backend/action/speaker/speak.py (hunk 3 above)
Comment (normanjaeckel) [reply openslides_backend/action/speaker/speak.py:15]: Done.

File: openslides_backend/action/speaker/speak.py (hunk 4 above)
Comment (normanjaeckel) [reply openslides_backend/action/speaker/speak.py:27]: Done.

**Comment by normanjaeckel** [comment]
Ready to merge.

File: openslides_backend/action/speaker/create_update_delete.py (hunk 2 above)
Comment (normanjaeckel) [reply openslides_backend/action/speaker/create_update_delete.py]: Changed for other reasons.

**Comment by normanjaeckel** [comment]
Of course we can use an ActionSet here but I prefer to see explicit actions that can be imported in some ListOfSpeakers actions directly (and not via action_map which is a little bit more implicit).

File: openslides_backend/action/list_of_speakers/delete_all_speakers.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    def prepare_dataset(self, payload: ActionPayload) -> DataSet:
+        data = []
+        for instance in payload:
+            list_of_speakers = self.fetch_model(
+                FullQualifiedId(self.model.collection, instance["id"]),
+                mapped_fields=["speaker_ids"],
+            )
+            if not list_of_speakers.get("speaker_ids"):
+                raise ActionException(
+                    f"List of speakers {instance['id']} has no speakers."
```
Comment (jsangmeister) [inline openslides_backend/action/list_of_speakers/delete_all_speakers.py:34]: Do we really want to throw an exception in this case? I think it's cleaner and more on course with the rest of the actions to just to nothing here. If you want to throw an exception, please add a test case for this.

File: tests/system/action/list_of_speakers/test_re_add_last.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
+from tests.system.action.base import BaseActionTestCase
```
Comment (jsangmeister) [inline tests/system/action/list_of_speakers/test_re_add_last.py:1]: Please add more tests, at least one for each possible exception thrown.

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Also, next time please open seperate PR's for each action. It's easier to review then. If you have dependencies between them you can base one on the other

**Pull request merged**